e2e/utils/constants/e2e.constants.ts

```diff
@@ -1,3 +1,7 @@
 export const HOMEPAGE_URL = '/';
 
 export const LOCAL_REPLICA_URL = 'http://127.0.0.1:4943';
+
+export const ABOUT_MODALS_VIEWPORT_WIDTH = 800;
+export const ABOUT_WHAT_MODAL_VIEWPORT_HEIGHT = 930;
+export const ABOUT_HOW_MODAL_VIEWPORT_HEIGHT = 1600;
```

Comment:
out of curiosity, why did you define `ABOUT_WHAT_MODAL_VIEWPORT_HEIGHT` and `ABOUT_HOW_MODAL_VIEWPORT_HEIGHT` as external constants and not directly in the test? will they be re-used in the future? or are already used somewhere I missed?

Reply:
Indeed, moved the height values to the related files. Though, the width const is still in `e2e.constants.ts` just so it can be re-used by both modals.